packages/frontend-shared/src/composables/useCollapsibleTree.ts

```diff
@@ -81,9 +86,9 @@ function buildTree (rawNode, options, acc: UseCollapsibleTreeNode[] = [], depth
   return acc
 }
 
-export function useCollapsibleTree (tree, options: UseCollapsibleTreeOptions = {}) {
+export function useCollapsibleTree <T extends RawNode<T>> (tree: T, options: UseCollapsibleTreeOptions = {}) {
   options.expandInitially = options.expandInitially || true
-  const collapsibleTree = buildTree(tree, options)
+  const collapsibleTree = buildTree<T>(tree, options)
 
   const expand = (matches?) => {
```

Comment:
I don't see anywhere in the code base that `expand` or `reveal` is called with an argument - what is `matches`? Is this used?

Reply:
I don't see a use for it but @JessicaSachs might have an opinion, maybe for future functionality?